requirements.txt

```diff
@@ -2,6 +2,8 @@ dogapi==1.2.1
 django>=1.8,<1.9
 django-extensions==1.5.9
 django-model-utils==2.3.1
-djangorestframework>=3.1,<3.2
+# Use the same cherry-picked repo as edx-platform
+#djangorestframework>=3.1,<3.2
```

Comment:
Yes, that's how it was done in edx-platform so I just copied it over. I can remove it if you prefer.